src/libs/actions/Policy.js

```diff
@@ -752,6 +753,73 @@ function clearAddMemberError(policyID, memberEmail) {
 }
 
 /**
+* Checks if we have any errors stored within the POLICY_MEMBER_LIST.  Determines whether we should show a red brick road error or not
+ * Data structure: {email: {role:'bla', errors: []}, email2: {role:'bla', errors: [{1231312313: 'Unable to do X'}]}, ...}
+ * @param {Object} policyMemberList
+ * @returns {Boolean}
+ */
+function hasPolicyMemberError(policyMemberList) {
+    return _.some(policyMemberList, member => !_.isEmpty(member.errors));
+}
+
+/**
+ * @param {String} value
+ * @returns {String}
+ */
+function capitalizeFirstLetter(value) {
+    return value.charAt(0).toUpperCase() + value.slice(1);
+}
+
+/**
+ * Generate a policy name based on an email and policy list.
+ * @param {String} email
+ * @param {Array} polices
+ * @returns {String}
+ */
+function generateDefaultWorkspaceName(email, polices = null) {
+    let userPolicies = null;
+    if (!polices) {
+        userPolicies = allPolicies;
+    } else {
+        userPolicies = polices;
+    }
+    const emailParts = email.split('@');
+    let defaultWorkspaceName = '';
+    if (!emailParts || emailParts.length !== 2) {
+        return defaultWorkspaceName;
+    }
+    const username = emailParts[0];
+    const domain = emailParts[1];
+    if (_.includes(PUBLIC_DOMAINS, domain.toLowerCase())) {
+        defaultWorkspaceName = `${capitalizeFirstLetter(username)}'s Workspace`;
+    } else {
+        defaultWorkspaceName = `${capitalizeFirstLetter(domain.split('.')[0])}'s Workspace`;
+    }
```

Comment:
lets make sure to add a line break between the const and the start of if statements